core/store/src/db.rs

```diff
@@ -85,13 +129,22 @@ pub enum DBCol {
     ColReceiptIdToShardId = 27,
     _ColNextBlockWithNewChunk = 28,
     _ColLastBlockWithNewChunk = 29,
-    /// Map each saved peer on disk with its component id.
+    /// Network related: Map each saved peer on disk with its component id (a.k.a. nonce).
+    /// - *Rows*: peer_id
+    /// - *Column type*:  u64
     ColPeerComponent = 30,
-    /// Map component id with all edges in this component.
+    /// Network related: Map component id  (a.k.a. nonce) with all edges in this component.
+    /// These are all the edges that were purged and persisted to disk at the same time.
+    /// - *Rows*: nonce
+    /// - *Column type*: `Vec<near_network::routing::Edge>`
     ColComponentEdges = 31,
-    /// Biggest nonce used.
+    /// Network related: Biggest component id (a.k.a nonce) used.
+    /// - *Rows*: single row (empty row name)
+    /// - *Column type*: u64
     ColLastComponentNonce = 32,
-    /// Transactions
+    /// Map of transactions (key is transaction hash)
```

Comment:
```suggestion
    /// Map of transactions
```

We don't explain that row=key anywhere else.

Reply:
done